modules/rosa-hcp-sts-creating-a-cluster-ext-auth-provider-cli.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
```

Comment:
@lucasponce @anjaltelang @zanetworker we had discussed documenting with Entra ID, which was the purpose of this ticket. Should I pivot to the KeyCloak and purge all the Entra ID items? This would likely push this PR back a bit, and I would need someone to explain how to use KeyCloak.